### PR TITLE
Update installation docs for third party dependencies

### PR DESCRIPTION
**Pull request by meatballs**

### CODE REVIEW

File: docs/guides/installation.rst [hunk 1]
```diff
@@ -1,64 +1,13 @@
...
-
-* Finally, push those changes to your copy of the app at Anvil:
+   * In the buttons to the right of 'Add a dependency', click the 'Third Party' button
+   * Enter the id of the Anvil-Extras app: C6ZZPAPN4YYF5NVJ=UGGCKFPRVZ7ELJH6RRZTHV6Y
+   * Hit enter and ensure that the library appears in your list of dependencies
+   * Select whether you wish to use the 'Development' or 'Published' version
 
-.. code-block::
+For the published version, the dependency will be automatically updated as new versions are released.
+On the development version, the update will occur whenever we merge new changes into the library's code base.
```
Comment (s-cork) [inline docs/guides/installation.rst:12]: Is it worth adding a note that - while we won't intentionally break your code the development branch should be considered unstable and should not be used in production.

File: docs/guides/installation.rst [hunk 2]
```diff
@@ -1,64 +1,13 @@
...
-
-.. code-block::
-
-    git fetch github
-    git reset --hard github/main
-
-* Finally, push those changes to your copy of the app at Anvil:
+   * In the buttons to the right of 'Add a dependency', click the 'Third Party' button
+   * Enter the id of the Anvil-Extras app: C6ZZPAPN4YYF5NVJ=UGGCKFPRVZ7ELJH6RRZTHV6Y
+   * Hit enter and ensure that the library appears in your list of dependencies
```
Comment (s-cork) [inline docs/guides/installation.rst:8]: ha yes hitting enter threw me for a second the first time I tried this!

**Review by s-cork — APPROVED** [review]
looks good to me. One suggestion about adding a comment for the dev vs prod branch

File: docs/guides/installation.rst (hunk 2 above)
Comment (meatballs) [reply docs/guides/installation.rst:8]: me too. That's there for my own benefit when I inevitably forget!

File: docs/guides/installation.rst (hunk 1 above)
Comment (meatballs) [reply docs/guides/installation.rst:12]: incoming...

**Comment by s-cork** [comment]
We should change the README too

**Pull request merged**